x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/InferenceFeatures.java

```diff
@@ -103,7 +104,8 @@ public Set<NodeFeature> getTestFeatures() {
                 TEXT_SIMILARITY_RERANKER_SNIPPETS,
                 ModelStats.SEMANTIC_TEXT_USAGE,
                 SEARCH_USAGE_EXTENDED_DATA,
-                TEXT_SIMILARITY_RANK_DOC_EXPLAIN_CHUNKS
+                TEXT_SIMILARITY_RANK_DOC_EXPLAIN_CHUNKS,
+                INFERENCE_AUTH_POLLER_PERSISTENT_TASK
```

Comment:
It's not necessary to add the `NodeFeature` to both `getFeatures()` and `getTestFeatures()` because right below this, we add everything from `getFeatures()` to the set returned by `getTestFeatures()`.